files/enc.rb

```diff
@@ -115,11 +115,10 @@ class NodeRetrievalError < StandardError; end
   end
 end
 
-def parse_file(filename, mac_address_workaround = false)
+def parse_file(filename)
   case File.extname(filename)
   when '.yaml'
     data = File.read(filename)
-    quote_macs!(data) if mac_address_workaround && YAML.safe_load('22:22:22:22:22:22').is_a?(Integer)
```

Comment:
As long as system ruby (`#!/usr/bin/env ruby`) is 2.5 or later, `YAML.safe_load('22:22:22:22:22:22')` won't be an `Integer`, so it becomes irrelevant whether the incoming yaml data is quoting mac addresses or not.